src/exercises/exerciseTypes/BottomInput.js

```diff
@@ -125,6 +132,8 @@ export default function BottomInput({
       // we give them a Hint, mainly for audio exercises.
       updatedMessageToAPI = messageToAPI + "H";
       setDistanceToCorrect();
+    } else if (isOneWordCorrect) {
```

Comment:
I am pretty sure that this won't run as the setter it's only run at the end of the code - line 118 needs to be stored in a temporary value that is used to compute the logic and not the React state.